remote_write/cases/labels.go

```diff
@@ -109,3 +109,20 @@ func NameLabelTest() Test {
 		},
 	}
 }
+
+// HonorLabels exports a single, constant metric with a job label
+// and checks that we receive metrics a exported_job label.
+func HonorLabelsTest() Test {
+	return Test{
+		Name: "HonorLabels",
+		Metrics: staticHandler([]byte(`
+# HELP test A gauge
+# TYPE test gauge
+test{job="original"} 1.0
```

Comment:
Does it make sense to test the same with `instance` as well?

Maybe also add a second label to verify that not everything get a prefix of `exported_`?

Reply:
> Does it make sense to test the same with instance as well?

Yeah can do! 

> Maybe also add a second label to verify that not everything get a prefix of exported_?

We test that labels are propagated correctly in other tests.